src/main/java/seedu/tasklist/model/task/Time.java

```diff
@@ -12,7 +12,7 @@
  * Represents the deadline of an assignment in the assignment list.
  * Guarantees: immutable; is valid as declared in {@link #isValidDeadline(String)}
  */
-public class Deadline {
+public class Time {
```

Comment:
The JavaDocs of the Time class should be changed to match the new name. 

Maybe something like:
"Represents the time found in a task in ProductiveNUS
Guarantees: immutable; is valid as decalred in {@link #isValidTime(String)"}

It would be ideal if isValidDeadline is changed to isValidTime as well, though if only the assignment class uses isValidDeadline, then it may be fine to leave the name as it is.

Reply:
okay thank you!!